java/org/contikios/cooja/plugins/TimeLine.java

```diff
@@ -604,6 +616,8 @@ private void zoomFinish (final double zoomDivisor,
     if (zoomDivisor != currentPixelDivisor) {
       logger.info("Zoom level: adjusted out-of-range " + zoomDivisor + " us/pixel");
     }
+    
+    if (note != "")
     logger.info("Zoom level: " + currentPixelDivisor + " microseconds/pixel " + note);
```

Comment:
I thought I removed the noise in commit 83c13a9b, just remove this line instead of making it conditional.